### PR TITLE
NHUB-86 ADD: Make company name unique

### PR DESCRIPTION
**Pull request by dev-tsoni**

### CODE REVIEW

**Comment by dev-tsoni** [comment]
> is it case sensitive? and pls write a test

No, it is not case sensitive. Is it supposed to be?

**Comment by MarkLark86** [comment]
> > is it case sensitive? and pls write a test
> 
> No, it is not case sensitive. Is it supposed to be?

I would assume so. It doesn't make sense to have two companies "Monkey Press" and "monkey press"

**Comment by dev-tsoni** [comment]
> > > is it case sensitive? and pls write a test
> > 
> > 
> > No, it is not case sensitive. Is it supposed to be?
> 
> I would assume so. It doesn't make sense to have two companies "Monkey Press" and "monkey press"

Yup, still working on it.

File: newsroom/companies/views.py [hunk 1]
```diff
@@ -63,7 +64,7 @@ def get_errors_company(company):
     if not company.get('name'):
         return jsonify({'name': gettext('Name not found')}), 400
 
-    if get_resource_service('companies').find_one(req=None, name=company['name']):
+    if company_exists(company['name']):
```
[the remark this answers is not in the document]
Comment (petrjasek) [reply newsroom/companies/views.py:67]: I've made a PR to your fork where that index issue is fixed:
https://github.com/dev-tsoni/newsroom-core/pull/1

**Comment by petrjasek** [comment]
included in https://github.com/superdesk/newsroom-core/pull/100

**Pull request closed without merging**